spec/features/registration_spec.rb

```diff
@@ -0,0 +1,86 @@
+feature 'registration' do
+  let(:email)     { Faker::Internet.unique.email }
+  let(:password1) { '1234567890' }
+  let(:password2) { '987654321' }
+
+  context 'signup form' do
+    before { visit '/' }
+
+    scenario 'User registration' do
+      signup(email, password1, password1)
+
+      expect(page).to have_css('.alert')
```

Comment:
it would be nice to add the text of that alert-message. Please add the text for every error/alert-message in every test